### ISSUE TITLE
Is it a proxy or redirector?

### ISSUE THREAD

**Issue by inikolaev** [lamhieu-vk/server-redirect#1]
I'm confused - is that a proxy server, not a redirector? From a redirector I would expect to return HTTP response with 301/302 status codes and `Location` header.

This server seems to be a proxy, not a redirector, correct me if I'm wrong.

**Comment by lh0x00** [comment]
@inikolaev  yep, the name of package is wrong, it should be a proxy instead of redirector.
you can suggest me another name?

**Comment by inikolaev** [comment]
`server-proxy`? If it's still available as a package name. :)
But would be nice if you could mention that somewhere in documentation, because examples provide no clue about whether it's proxy or not.
